packages/core/.storybook/middleware.js

```diff
@@ -0,0 +1,10 @@
+const { createProxyMiddleware } = require('http-proxy-middleware')
+
+module.exports = function (app) {
```

Comment:
In `preview-body.html` wordt het development iframe van de Stencil compiler ingeladen. Stencil gebruikt het iframe om een HMR/live reload te realiseren. Het iframe heeft beperkingen mbt cross origin requests.

Deze proxy zorgt ervoor dat het een same origin request wordt, de backend van storybook zet het request door naar het Stencil backend proces